src/common/transformations/src/transformations/op_conversions/convert_subtract.cpp

```diff
@@ -4,75 +4,72 @@
 
 #include "transformations/op_conversions/convert_subtract.hpp"
 
-#include <memory>
 #include <ngraph/opsets/opset1.hpp>
 #include <ngraph/pattern/op/wrap_type.hpp>
 #include <ngraph/rt_info.hpp>
-#include <vector>
+#include <openvino/core/validation_util.hpp>
 
 #include "itt.hpp"
+#include "transformations/rt_info/dequantization_node.hpp"
 
-ngraph::pass::ConvertSubtract::ConvertSubtract() {
+using namespace ngraph;
+
+static bool convert_subtract(const std::shared_ptr<Node>& node) {
+    auto sub = std::dynamic_pointer_cast<opset1::Subtract>(node);
+    if (!sub) {
+        return false;
+    }
+
+    if (ov::is_dequantization_node(sub)) {
+        return false;
+    }
+
+    if (!sub->get_input_element_type(1).is_signed()) {
+        return false;
+    }
+
+    std::shared_ptr<Node> neg =
+        std::make_shared<opset1::Multiply>(sub->input_value(1),
+                                           opset1::Constant::create(sub->get_input_element_type(1), Shape{}, {-1}));
+    NodeVector new_nodes;
```

Comment:
Consider use [NodeRegistry](https://github.com/openvinotoolkit/openvino/blob/f6d6f5629fa5679717de31e7a8a8506a77550ed0/src/core/include/openvino/pass/graph_rewrite.hpp#L22) class here.
Then you can make/add nodes by this registry and copy `runtime_info` as 
```cpp
 copy_runtime_info(sub, new_nodes.get());
```